### ISSUE TITLE
DetailsList: Edge + narrator doesnot say anthing about the sortable detailslist column header

### ISSUE THREAD

**Issue by shsengar** [OfficeDev/office-ui-fabric-react#11131]
#10957 # Describe the issue:
Edge+ Narrator is not providing info about the column header if it is sortable or not.

### Please provide a reproduction of the issue in a codepen:
1) Open Edge Browser.
2) Go to https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist
3) Start narrator as screen reader.
4) Use Caps+Right Arrow key to navigate to "Name" Column Header(which is sortable). Observe what narrator is saying

![image](https://user-images.githubusercontent.com/57530569/68481990-9187d100-025e-11ea-9159-51113d4e8971.png)

#### Actual behavior:

Narrator is not providing information if the column is functionally sortable.

#### Expected behavior:
Narrator should announce the header information along with sorting order information. Visually impaired users will find difficulty to know the sorting functionality if its available.

**Comment by JasonGore** [comment]
Duplicate of #10168, #6646, and #6011. Please search for duplicates before creating issues.
